### DISCUSSION TITLE
[Offline] Firefox Doesn't allow passwords over HTTP

### DISCUSSION THREAD

**Discussion by arky** [Terrastories/terrastories#758] [Temporarily Archived Tickets]
Terrastories logins will work in Firefox (85.0/Linux)

**Describe the bug**

**To Reproduce**
1. Go to 'terrastories login page'
2. Enter the username and password.
3. Firefox will not allow you to login and show you insecure password login 

**Desktop (please complete the following information):**
 - OS: gnu/Linux 
 - Browser Firefox
 - Version 85

**Additional context**
See https://support.mozilla.org/en-US/kb/insecure-password-warning-firefox
https://blog.mozilla.org/tanvi/2016/01/28/no-more-passwords-over-http-please/

**Comment by rudokemper** [comment]
Hi @arky,

We just added SSL to the Terrastories server you have access to. It should be resolved now. Can you check it out from your end and see?

**Comment by arky** [comment]
Great @rudokemper  this works for online deployments. 

But SSL will not work for offline-only deployments of Terrastories

**Comment by arky** [comment]
@rudokemper The solution for this issue is use tools like mkcert or OpenSSL toolchain to generate your own certificates.  

https://github.com/FiloSottile/mkcert
